hstream/test/HStream/Common.hs

```diff
@@ -0,0 +1,160 @@
+{-# LANGUAGE GADTs               #-}
+{-# LANGUAGE OverloadedStrings   #-}
+{-# LANGUAGE PatternSynonyms     #-}
+{-# LANGUAGE RecordWildCards     #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module HStream.Common where
```

Comment:
What does this "common" do?

Reply:
For the use of testing query related gRPC apis...
Reference: https://github.com/hstreamdb/hstream/pull/364